alf/environments/process_environment.py

```diff
@@ -214,6 +252,8 @@ def __getattr__(self, name):
             Value of the attribute.
         """
         assert self._conn, "Run ProcessEnvironment.start() first"
+        if self._fast:
+            self._penv.call()
```

Comment:
This is a part that is unclear to me. By design, if we delegate all the `main process` / `child process` communication to shared memory buffers in c++, why do we still need to use `Pipe` in fast mode? To put my question in a better way, isn't it possible to make the design such that Pipe is no longer needed in fast mode?

Reply:
Updated the description about "call" job in the PR description.

Reply:
Very clear now. Thank you!